pages/ai/content/ai-basics/_ai-basics-carousel.html

```diff
@@ -1,27 +1,22 @@
 {% set slides = [
   { "alt": "Tokenization Visualization","title": "Tokenization", "description": "The LLM starts by breaking down the input text, or 'prompt', into smaller pieces known as tokens.", "image": "/img/ai/carousels/how-llm-works/tokenization.png"},
-  { "alt": "Embedding Visualization","title": "Embedding", "description": "Next, each token is converted into a numerical representation through a process called embedding", "image": "/img/ai/carousels/how-llm-works/embedding.png"},
+  { "alt": "Embedding Visualization","title": "Embedding", "description": "Next, each token is converted into a numerical representation through a process called embedding.", "image": "/img/ai/carousels/how-llm-works/embedding.png"},
   { "alt": "Self-attention Visualization","title": "Self-attention", "description": "The model then adds additional information to these vectors to account for the order of words in the input. Self attention mechanisms allow the model to create context-aware representations of each word.", "image": "/img/ai/carousels/how-llm-works/self-attention.png"},
   { "alt": "Decoding Visualization","title": "Decoding", "description": "The LLM Decoder will take the output and use it to predict the next token in the sequence.", "image": "/img/ai/carousels/how-llm-works/decoding.png"},
   { "alt": "Output Visualization","title": "Output", "description": "Finally, the model generates a response.", "image": "/img/ai/carousels/how-llm-works/output.png"}
 ]
 %}
 
-<div class="carousel rounded-box carousel-center max-w-3xl mt-4 mb-4 p-4 space-x-4 rounded-box border-2 border-gray-900 shadow-md shadow-primary">
+<div class="whitespace-nowrap overflow-x-scroll flex py-6">
   {% for slide in slides %}
-  <div id="item{{loop.index}}" class="carousel-item card card-compact bg-neutral w-72 p-4">
-      <figure class="card-figure">
+  <div id="item{{loop.index}}" class="card card-compact bg-neutral w-72 shrink-0 grow-0 ml-6 first:ml-0">
+      <figure class="card-figure !mt-0 !mb-0">
```

Comment:
You might be able to just use `!my-0` instead of specifying `mt-*` and `mb-*` separately.
https://tailwindcss.com/docs/margin#add-vertical-margin

Not sure if you'd need to make this change in 4 different files (Ctrl+F says there are 4 matches for `!mt-`), which might be another nod to using DRYer [`macro`](https://mozilla.github.io/nunjucks/templating.html#macro)s.

Reply:
@pdehaan  - feel free to implement this and PR if you like. Otherwise will revisit macros when I have time.